takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -8,18 +8,79 @@
 {% block dropdown_desc %}Lorem ipsum{% endblock %}
 
 {% block dropdown_content %}
-<div class="row pl-4" style="width: 100%;">
+<div class="row pl-4 pb-5" style="width: 100%;">
     <div class="col-sm-4">
-      <p><a href="/about">About Takwimu</a></p>
-      <p><a href="/about/support-services">Support Services</a></p>
-      <p><a href="/contact-us">Contact Us</a></p>
-      <p class="mb-0"><a href="/dashboard">Dashboard (Admin)</a></p>
+      <ul style="opacity: 0.6;">
```

Comment:
Why use opacity here? If the linked design from #179 is still current, these links should be light blue and not faded.